lib/cisco_node_utils/cmd_ref/span_session.yaml

```diff
@@ -0,0 +1,64 @@
+# SPAN session
+---
+_template:
+  nexus:
+    get_command: "show running monitor all"
+
+all_sessions:
+  multiple:
+  get_context: ~
+  get_value: '/^monitor session (\d+)$/'
+
+create:
+  set_value: 'monitor session %s'
+
+description:
+  kind: string
+  get_context: ["monitor session <id>"]
+  get_value: '/^description (.*)/'
+  set_context: ["monitor session <id>"]
+  set_value: '<state> description <desc>'
+  default_value: ''
+
+destination:
+  kind: string
+  get_context: ["monitor session <id>"]
+  get_value: '/^destination interface <intf_name>$/'
+  set_context: ["monitor session <id>"]
+  set_value: '<state> destination interface <intf_name>'
+
+destroy:
+  set_value: 'no monitor session %s'
+
+session_id:
+  kind: int
+  default_value: false
+  get_value: '/^monitor session (\d+)$/'
+  set_value: '<state> monitor session <id>'
+
+shutdown:
+  kind: boolean
+  get_context: ["monitor session <id>"]
+  get_value: '/^(\S+) shutdown$/'
```

Comment:
How does this nvgen on your test device?  In my case I see the following.

**Default State**
```
# show running monitor all
monitor session 1 type local
  no description
  shut
# 
```
I would refactor the yaml entry as follows:

```yaml
  kind: boolean
  default_value: true
  get_context: ["monitor session <id>"]
  get_value: '/^no shut$/'
  set_context: ["monitor session <id>"]
  set_value: '<state> shutdown'
```
In this way, when you call `config_get('span_session', 'shutdown', id: @session_id)` it returns `true` if it's in the shutdown state, else false.